include/pinvoke_api.hpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
?

Reply:
Well, it is, under the MIT license. We fit to their header exactly. This is technically correct.